eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -130,28 +130,14 @@ jobs:
         displayName: Install dev-tool
 
       - download: current
-        artifact: ${{parameters.ArtifactName}}
+        condition: and(succeeded(), ne('{{ $parameters.Daily }}', 'true'))
       - pwsh: |
           $(Build.SourcesDirectory)/eng/common/scripts/Import-AzModules.ps1
 
           $subscriptionConfiguration = @"
             $(SubscriptionConfiguration)
           "@ | ConvertFrom-Json -AsHashtable;
 
-          $packageOverrides = @()
-          $packageOverrides += (dir env: | Where-Object { $_.Name.StartsWith("SMOKE_PACKAGE_") }).Value
```

Comment:
If we remove this we should also clean up the setting of the SMOKE_PACKAGE_ env variables below.